drush.services.yml

```diff
@@ -1,5 +1,6 @@
 services:
   responsive_image_effect.commands:
     class: \Drupal\responsive_image_effect\Commands\ResponsiveImageEffectCommands
+    arguments: ['@responsive_image_effect.responsive_image_service', '@entity_type.manager']
```

Comment:
Is this because drupal doesn't autowire anymore?

Reply:
@bendeeson yep tried autowiring and it didn't seem to work at all, not sure if Drupal is doing anything different from Symfony but I kept getting cannot autowire and/or not enough arguments errors. So looked at the big contrib modules like Webform and they don't autowire either. So not sure if something is broken with Drupal's autowire implementation.